playbooks/roles/airship-configure-ceph/templates/k8s-ses-storageclass-rook.yml.j2

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: {{ suse_airship_deploy_storage.pool }}
```

Comment:
nit: I would prefer if this would be static. For once I prefer less flexibility, because it makes the integration far easier. But we can keep it this way currently.

Reply:
I'm not sure what is the problem, if you mean the fact that we have one SC for Airship deployment and one for OSH then I agree. 
But I think I'd rather tackle it in some followup card because it's not affecting just SES-rook

Reply:
https://jira.suse.com/browse/SOC-10372

Reply:
Thanks!